packages/cashier/package.json

```diff
@@ -43,6 +43,7 @@
         "@deriv/deriv-api": "^1.0.15",
         "@deriv/hooks": "^1.0.0",
         "@deriv/p2p": "^0.7.3",
+        "@deriv/quill-icons": "^1.22.10",
```

Comment:
Hi @farrah-deriv,
IMO we should rely on legacy components package instead of quill-icon as this will just unnecessarily increase the bundle size for cashier package.
If you want to use it for some specific requirement, maybe you can include it under the `externals` object of webpack.config.js for cashier package.

Reply:
Thanks for your suggestion @aum-deriv. The legacy icon for this looks different that's why I opted to use the quill icon.